docs.yml

```diff
@@ -0,0 +1,3 @@
+title: Fluri
```

Comment:
out of curiosity should this be `fluri` or is this a dev portal convention to have it capitalized?